README.md

```diff
@@ -49,6 +49,17 @@ sed -i "s/<REPLACE ME>/GITHUB_NICK/g" boxes.d/99-local.yaml
 vagrant up centos7-devel
 ```
 
+In case you don't want to use vagrant, you can still use ansible playbooks and roles from this repo. This is useful if you have existing host, e.g. a beaker machine. Let's say you have host test.example.com on which you want to deploy devel environment, following should do the trick:
+
+```sh
+echo "[devel]\ntest.example.com" > local_inventory
+ansible-playbook --private-key=/home/ares/.ssh/id_rsa --user root --inventory local_inventory --extra-vars katello_devel_github_username=katello playbooks/devel.yml
+```
+
+In an example above I pointed ansible to use specific private key (overriding the value from ansible.cfg), I use root as ssh user and we set playbook variable so that we'll checkout code from katello user.
```

Comment:
Switching this to third person would help keep it in line with the rest of the README docs.